contracts/lbp/LBPWrapper.sol

```diff
@@ -16,13 +16,17 @@ import "openzeppelin-contracts-sol8/token/ERC20/IERC20.sol";
 import "../utils/interface/ILBPFactory.sol";
 import "../utils/interface/IVault.sol";
 import "../utils/interface/ILBP.sol";
+import "hardhat/console.sol"; //
```

Comment:
🙃

Reply:
Damn I even put the comments behind it so I wouldn't forget to remove it :)